docs/_sass/custom/custom.scss

```diff
@@ -887,3 +920,15 @@ div.highlighter-rouge{
 		color: $light-blue-grey;
 	}
 }
+@media (max-width:767px){
+	.text-delta + ul, #markdown-toc{
+		li, li a{
+			margin-bottom: 12px;
+		    font-size: 12px !important;
+		    display: block;
+		}
+		li a span{
+			font-size: 8px!important;
```

Comment:
do we need to use '!important' for all the places we change the font-size?

Reply:
Yes I only used important where needed because override things not working here so that's why important needed